.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+* @confluentinc/kafka-rest-eng
```

Comment:
Foundations team owns this:

* https://confluentinc.atlassian.net/wiki/spaces/Foundations/pages/2530804312/Ownership+of+rest-utils
* https://github.com/confluentinc/common-tools/blob/master/confluent/config/dev/projects.json#L3412

Reply:
https://github.com/orgs/confluentinc/teams?query=foundations